src/v/storage/log_manager.cc

```diff
@@ -429,6 +431,82 @@ ss::future<> log_manager::remove(model::ntp ntp) {
     co_await dispatch_topic_dir_deletion(topic_dir);
 }
 
+ss::future<> remove_orphan_partition_files(
+  ss::sstring topic_directory_path,
+  model::topic_namespace nt,
+  const log_manager::ntp_to_revision& topics_on_node,
+  model::revision_id& last_applied_revision) {
```

Comment:
nit: revision_id is trivially copyable, pass by value.

Reply:
fixed